Doppler.HtmlEditorApi/Controllers/TemplatesController.cs

```diff
@@ -0,0 +1,148 @@
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Doppler.HtmlEditorApi.DopplerSecurity;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.Extensions.Logging;
+using Doppler.HtmlEditorApi.Model;
+
+
+namespace Doppler.HtmlEditorApi.Controllers
+{
+    [Authorize]
+    [ApiController]
+    public class TemplatesController : Controller
+    {
+        [Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]
+        [HttpGet("/accounts/{accountName}/templates/{templateId}")]
+        public ActionResult<CampaignModel> GetTemplate(int templateId, string accountName)
```

Comment:
Very minor, but in general we are using the account name first.

```suggestion
        public ActionResult<CampaignModel> GetTemplate(string accountName, int templateId)
```